plugins/linux-capture/xcomposite-input.c

```diff
@@ -399,30 +381,6 @@ static enum gs_color_format gs_format_from_tex()
 	}
 }
 
-// These declarations are from libobs-opengl/gl-subsystem.h because we need to
-// handle GLX modifying textures outside libobs.
-struct fb_info;
-
-struct gs_texture {
-	gs_device_t *device;
-	enum gs_texture_type type;
-	enum gs_color_format format;
-	GLenum gl_format;
-	GLenum gl_target;
-	GLenum gl_internal_format;
-	GLenum gl_type;
-	GLuint texture;
-	uint32_t levels;
-	bool is_dynamic;
-	bool is_render_target;
-	bool is_dummy;
-	bool gen_mipmaps;
-
-	gs_samplerstate_t *cur_sampler;
-	struct fbo_info *fbo;
-};
-// End shitty hack.
-
 // XErrorHandler for glXCreatePixmap calls.
```

Comment:
We can drop the X error handler as well, its not being called in the new egl path.

Reply:
Good catch!